### DISCUSSION TITLE
Online trend shows only last 2 minutes - How to record long time?

### DISCUSSION THREAD

**Discussion by hometm** [BernhardSchlegel/BierBot-Bricks#26] [Q&A]
Online trend e.g. at manual mode shows only last 2 minutes. After zooming out, the history is not loaded. An automatically reload resets the trend view

![onlineTrend](https://user-images.githubusercontent.com/10582162/140655367-07798ec5-b0eb-4c72-b21e-4760acbfdbed.jpg)

**Comment by BernhardSchlegel** [comment]
Hey Hometm,

the manual mode is not supposed to track long time. Sounds like your looking for the [automatic mode](https://docs.bierbot.com/using-bricks/automatic-mode). Create a recipe and start that and you should be good to go!

Cheers,
Bernhard.
